action_server/pyproject.toml

```diff
@@ -26,13 +24,14 @@ uvicorn = "^0.23.2"
 pydantic = "^2.4.2"
 jsonschema = "^4.19.2"
 PyYAML = "^6"
-robocorp-actions = "^0.0.2"
+# robocorp-actions = "^0.0.2"
+robocorp-actions = { path = "../actions", develop = true }
```

Comment:
wip: release new package version and replace

Reply:
This is not needed (and should be reverted).

We have a command: `inv devinstall` which will temporarily do this change to install the env in devmode and then it'll switch back to the proper env aferwards.